configs/containers/docker-ubuntu-gcc-openmpi.yaml

```diff
@@ -30,13 +29,18 @@ spack:
       require: '%gcc'
       target: [x86_64]
       providers:
-        mpi: [openmpi@4.1.4]
+        mpi: [openmpi@4.1.5]
```

Comment:
just wanted to confirm this version change was intended, only because i didn't see it in the PR description.

Reply:
yes it was, thanks! I'll update the PR description